README.md

```diff
@@ -44,9 +44,10 @@ Another option to override the default values can use `--action_env` as in the e
 | MUST_GATHER_IMAGE     | quay.io/kubev2v/forklift-must-gather:latest      | The forklift must gather an image.                          |
 | MUST_GATHER_API_IMAGE | quay.io/kubev2v/forklift-must-gather-api:latest  | The forklift must gather image api.                         |
 | UI_IMAGE              | quay.io/kubev2v/forklift-ui:latest               | The forklift UI image.                                      |
-| UI_PLUGIN_IMAGE       | quay.io/kubev2v/forklift-console-plugin:latest   | The forklift OKD/OpenShift UI plugin image.                                      |
+| UI_PLUGIN_IMAGE       | quay.io/kubev2v/forklift-console-plugin:latest   | The forklift OKD/OpenShift UI plugin image.                 |
 | VALIDATION_IMAGE      | quay.io/kubev2v/forklift-validation:latest       | The forklift validation image.                              |
 | VIRT_V2V_IMAGE        | quay.io/kubev2v/forklift-virt-v2v:latest         | The forklift virt v2v image.                                |
+| OVIRT_IMAGEIO_IMAGE   | quay.io/kubev2v/ovirt-populator:latest           | The forklift ovirt populator image.                         |
```

Comment:
there's another image for the ovirt-imageio container (in [quay.io/ovirt](https://quay.io/repository/ovirt/ovirt-imageio)), how about OVIRT_POPULATOR_IMAGE?